vm/parser.ts

```diff
@@ -711,6 +861,17 @@ module J2ME {
       }
     }
 
+    getClassNameSlow(): string {
+      return this._name || (this._name = ByteStream.readString(this.utf8Name));
+    }
```

Comment:
> This would be a very unexpected result. At this point, both the getter and the function call are effectively the same thing, and are most likely optimized the same way. I was really surprised to see that the function call was slower, so I tried it in the jsshell. In the shell there is no difference between getters and function calls.

Ah, indeed, I get the same results. I suspect our use case resembles the few objects/many accesses scenario, where the difference between the memoizer and (either) getter is insignificant. But I'd still rather not depend on that. So you've convinced me that the getter approach is better.

> I also tried it in an html page and the results were the same as the shell. This suggests that jsfiddle is not to be trusted, ... now I'm curious why it's slower there, I suspect they may be using eval.

Indeed, I'm also curious. cc: @zalun for his thoughts.